src/main/java/zipkin/sparkstreaming/adjuster/BinaryAnnotationTrimAdjusterConfiguration.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2017 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package zipkin.sparkstreaming.adjuster;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import zipkin.sparkstreaming.Adjuster;
+
+@Configuration
+public class BinaryAnnotationTrimAdjusterConfiguration {
```

Comment:
I think this could hit harder actually using spring

```java
@Configuration
public class TagTruncatorConfiguration {
  /** This lets you override the max length via commandline, like --tagtruncator.max-length=255 */
  @Bean
  Adjuster tagTruncator(@Value("tagtruncator.max-length:1024") int maxLength) {
    return new TagTruncator(maxLength);
  }
}
```